internal/pkg/model/receiver.go

```diff
@@ -1,5 +1,23 @@
 package model
 
+import (
+	"time"
+
+	"github.com/c2h5oh/datasize"
+)
+
+type ImportCondition struct {
+	Count int               `json:"count" validate:"min=1,max=10000000"`
+	Size  datasize.ByteSize `json:"size" validate:"min=100B,max=50MB"`
+	Time  time.Duration     `json:"time" validate:"min=30s,max=24h"`
+}
+
+type Export struct {
+	ID               string            `json:"exportId" validate:"required,min=1,max=48"`
+	Name             string            `json:"name" validate:"required,min=1,max=40"`
+	ImportConditions []ImportCondition `json:"importConditions" validate:"required"`
```

Comment:
Tu pribudne este `mapping` ako som komentoval v Confluence, ale to asi teraz neriesis.

Reply:
Dodělám až pak v rámci `CreateExport`